### PR TITLE
minor fix: use configured saltGenerator if provided for createDecoy

### PR DESCRIPTION
**Pull request by ragnika**

### CODE REVIEW

File: src/common.ts [hunk 1]
```diff
@@ -106,7 +106,7 @@ export const packSDJWT: PackSDJWT = async (claims, disclosureFrame, hasher, opti
       }
     }
 
-    const decoys = createDecoy(disclosureFrame[SD_DECOY_COUNT], hasher);
+    const decoys = createDecoy(disclosureFrame[SD_DECOY_COUNT], hasher, options?.generateSalt);
```
Comment (linasi) [inline src/common.ts:109]: @ragnika  is it possible that `generateSalt` is not provided?
I see that it is required for `issueSDJWT`, shouldn't it follow the same logic here?

File: src/helpers.ts [hunk 2]
```diff
@@ -309,15 +309,15 @@ export const combineSDJWT = (jwt: string, disclosures: string[], kbjwt?: string)
   return combined;
 };
 
-export const createDecoy = (count: number, hasher: Hasher): string[] => {
+export const createDecoy = (count: number, hasher: Hasher, saltGenerator: SaltGenerator = generateSalt): string[] => {
```
Comment (linasi) [inline src/helpers.ts:312]: @ragnika if `saltGenerator` is required we don't need to add any default values.
If we keep this logic - please write a test or two to cover this part.

**Review by linasi — CHANGES_REQUESTED** [review]
Looks good. Take a look at my questions.

File: src/common.ts (hunk 1 above)
Comment (ragnika) [reply src/common.ts:109]: @linasi `issueSDJWT` shouldn't require the generateSalt, I'll update the type to remove that.
It was in the options arg for that reason.

**Comment by ragnika** [comment]
@linasi added the tests and updated the README on the optional generateSalt function.

Please merge if you think it's okay

**Pull request merged**